src/base/emu-i386/simx86/trees.c

```diff
@@ -1280,6 +1294,10 @@ int InvalidateNodeRange(int al, int len, unsigned char *eip)
   return cleaned;
 }
 
+int InvalidateNodeRange(int al, int len, unsigned char *eip)
+{
+  return InvalidateNodeRangeP0(al, len, eip, (unsigned)-1);
```

Comment:
That's weird, we already pass
NULL as *eip in most cases, which
was already annoying enough.